<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_DocumentSetup</class>
 <widget class="QWidget" name="Prefs_DocumentSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Document Setup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>686</width>
        <height>713</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Page Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="pageSizeLabel">
             <property name="text">
              <string>Size:</string>
             </property>
             <property name="buddy">
              <cstring>pageSizeComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="pageSizeComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="pageOrientatioLabel">
             <property name="text">
              <string>Orientation:</string>
             </property>
             <property name="buddy">
              <cstring>pageOrientationComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="pageOrientationComboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="pageUnitsLabel">
             <property name="text">
              <string>Units:</string>
             </property>
             <property name="buddy">
              <cstring>pageUnitsComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="pageUnitsComboBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="pageWidthLabel">
             <property name="text">
              <string>Width:</string>
             </property>
             <property name="buddy">
              <cstring>pageWidthSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="ScrSpinBox" name="pageWidthSpinBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="pageHeightLabel">
             <property name="text">
              <string>Height:</string>
             </property>
             <property name="buddy">
              <cstring>pageHeightSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="ScrSpinBox" name="pageHeightSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Page Layouts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QRadioButton" name="singlePageRadioButton">
           <property name="text">
            <string>Single Page</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QRadioButton" name="doublePageRadioButton">
           <property name="text">
            <string>Double Page</string>
           </property>
           <attribute name="buttonGroup">
            <string>pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QRadioButton" name="threeFoldRadioButton">
           <property name="text">
            <string>3-Fold</string>
           </property>
           <attribute name="buttonGroup">
            <string>pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QRadioButton" name="fourFoldRadioButton">
           <property name="text">
            <string>4-Fold</string>
           </property>
           <attribute name="buttonGroup">
            <string>pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="layoutFirstPageIsLabel">
           <property name="text">
            <string>First Page is:</string>
           </property>
           <property name="buddy">
            <cstring>layoutFirstPageIsComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="layoutFirstPageIsComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Margins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="NewMarginWidget" name="marginsWidget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="applyMarginsToAllPagesCheckBox">
         <property name="text">
          <string>Apply changes to all pages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="appleMarginsToAllMasterPagesCheckBox">
         <property name="text">
          <string>Apply changes to all master pages</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bleeds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="NewMarginWidget" name="bleedsWidget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>NewMarginWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/newmarginwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="pageLayoutButtonGroup"/>
 </buttongroups>
</ui>
